Service/LoadUIImage.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+import UIKit
+
+extension UIImageView {
+
```

Comment:
I have said before. You should choose 1 of these functions to get images from URL and delete another